data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -1293,7 +1293,7 @@
 
         [message]
             speaker=Abhai
-            message= _ "This is the Great Rod of Justice, my boy. It was an ancient artifact even when I was young. They say it was crafted by the Great Gods themselves, and given to the first true Ruler of Men to ensure peace, harmony and above all — justice. For hundreds of years it did just that, for during the time it was wielded by men, the ruling class never became corrupt, the people never lacked justice and neither evil nor wars troubled the land."
+            message= _ "This is the Great Rod of Justice, my boy. It was an ancient artifact even when I was young. They say it was crafted by the Great Gods themselves, and given to the first true Ruler of Men to ensure peace, harmony and above all — justice. For hundreds of years it did just that, for during the time it was wielded, the ruling class never became corrupt, the people never lacked justice, and neither evil nor wars troubled the land."
```

Comment:
While it might seem redundant to have two mentions of men - as a race, not a gender - I think the point is to emphasise that this story was specific to humanity. I suppose I don't mind either way.

Reply:
I think the change is fine since the previous sentence had "Men" in it too.